# bismark

[Back to application catalog](../app_catalog.md)

## Description

A tool to map bisulfite converted sequence reads and determine cytosine methylation states

## Homepage

[https://www.bioinformatics.babraham.ac.uk/projects/bismark](https://www.bioinformatics.babraham.ac.uk/projects/bismark){:target="_blank"}

## Available Versions on RCAC Clusters

|Cluster|Versions|
|---|---|
**ANVIL**|0.23.0

## Module

You can load the module by:

```bash
module load bismark
```

!!! note "Note for using `bismark`"
    Run `module spider bismark` beforehand to check if this version requires any prerequisite modules.
